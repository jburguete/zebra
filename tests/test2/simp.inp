[TITLE]
C:\Mario\Violada

[JUNCTIONS]
;ID              	Elev        	Demand      	Pattern         
 702             	378         	0           	                	;
 921             	375         	0           	                	;
 226             	371.48      	211        	                	;

[RESERVOIRS]
;ID              	Head        	Pattern         
 0               	435       	                	;

[PIPES]
;ID              	Node1           	Node2           	Length      	Diameter    	Roughness   	MinorLoss   	Status
 54              	921             	226             	365         	1176        	0.01        	0           	Open  	;
 57              	921             	702             	266         	285         	0.01        	0           	Open  	;
 9               	0               	921             	93          	1176        	0.01        	0           	Open  	;

[COORDINATES]
;Node            	X-Coord         	Y-Coord
 0               	1974.80         	4364.11         
 702             	1997.54         	4075.93         
 921             	1993.45         	4368.49         
 226             	2495.00         	4445.00         

[OPTIONS]
 Units              	LPS
 Headloss           	D-W
 Specific Gravity   	1
 Viscosity          	1
 Trials             	40
 Accuracy           	0.001
 CHECKFREQ          	2
 MAXCHECK           	10
 DAMPLIMIT          	0
 Unbalanced         	Continue 10
 Pattern            	1
 Demand Multiplier  	1.0
 Emitter Exponent   	0.5
 Quality            	None mg/L
 Diffusivity        	1
 Tolerance          	0.01

[BACKDROP]
 DIMENSIONS      	0               	0               	4707.2          	7259.33         
 UNITS           	None            
 FILE            	
 OFFSET          	0.00            	0.00            

[REPORT]
NODES 226
FLOW YES
VELOCITY PRECISION 4
F-FACTOR PRECISION 4
VELOCITY ABOVE 3.0


[END]
