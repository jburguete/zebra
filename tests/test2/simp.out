  Page 1                                    Fri Jul  2 05:54:31 2021

  ******************************************************************
  *                           E P A N E T                          *
  *                   Hydraulic and Water Quality                  *
  *                   Analysis for Pipe Networks                   *
  *                         Version 2.00.12                        *
  ******************************************************************
  
  C:\Mario\Violada 
   
      Input Data File ................... simp.inp
      Number of Junctions................ 3
      Number of Reservoirs............... 1
      Number of Tanks ................... 0
      Number of Pipes ................... 3
      Number of Pumps ................... 0
      Number of Valves .................. 0
      Headloss Formula .................. Darcy-Weisbach
      Hydraulic Timestep ................ 1.00 hrs
      Hydraulic Accuracy ................ 0.001000
      Status Check Frequency ............ 2
      Maximum Trials Checked ............ 10
      Damping Limit Threshold ........... 0.000000
      Maximum Trials .................... 40
      Quality Analysis .................. None
      Specific Gravity .................. 1.00
      Relative Kinematic Viscosity ...... 1.00
      Relative Chemical Diffusivity ..... 1.00
      Demand Multiplier ................. 1.00
      Total Duration .................... 0.00 hrs
      Reporting Criteria:
         Selected Nodes
         No Links
         with Velocity above 3.00 m/s
   
  Analysis begun Fri Jul  2 05:54:31 2021

   
  Node Results:
  ----------------------------------------------
                     Demand      Head  Pressure
  Node                  L/s         m         m
  ----------------------------------------------
  226               2.110000e+02   4.349886e+02   6.350856e+01
   
   
  Link Results:
  --------------------------------------------------------
                       Flow  Velocity  Headloss  F-Factor
  Link                  L/s       m/s    /1000m          
  --------------------------------------------------------
  54                 211.00    0.1943      0.02    0.0153
  57                  -0.00    0.0000      0.00    0.0000
  9                  211.00    0.1943      0.03    0.0154
   
  Analysis ended Fri Jul  2 05:54:31 2021
